formicarium/formicarium/DiffRobot.py

```diff
@@ -43,6 +44,8 @@ def __init__(self,name:str, wheel_radius: float, wheel_base: float, start_x: flo
         self.wheel_base = wheel_base
         self.wheel_radius = wheel_radius
         self.rect = self.image.get_rect(center=(self.x, self.y))
+        self.hitbox = pygame.Rect(0, 0, 69.5, 69.5)
```

Comment:
```suggestion
        self.hitbox = Rect(0, 0, 69.5, 69.5)
```
here you can then just use Rect